collect_app/src/main/java/org/odk/collect/android/gdrive/sheets/SheetsHelper.java

```diff
@@ -81,6 +83,24 @@ public void resizeSpreadSheet(String spreadsheetId, int sheetId, int columnSize)
         sheetsAPI.batchUpdate(spreadsheetId, requests);
     }
 
+    public void updateSpreadsheetLocale(String spreadsheetId) {
+        SpreadsheetProperties sheetProperties = new SpreadsheetProperties()
```

Comment:
Feels like it would be good to have a comment here to explain why this is happening.

Reply:
Maybe... personally I don't like comments and prefer using git history but if you think it's valuable I can do that.

Reply:
I understand the sentiment but I don't think our codebase is very well suited to this. It's really hard to find relevant commits when code has been refactored and in practice we move things around a lot. We'd also have to be more disciplined about rebasing to avoid noisy commits like "Refactored for readability" or "Fixed tests" or things like that which don't convey much information. As it is, we often have to go through many blame levels to find an explanation for a change. I don't find the Android Studio history tools particularly helpful in those cases or easy to access so I go to GitHub to find this type of information. I use a browser plugin to try to follow moved files but it doesn't always work and is of course not helpful if a method changed classes. If you have a tool or strategy that lets you get to reasoning quickly even through refactors, I'd love to see a demo. That would certainly change my thinking on this.

In this case, I think a simple note like "Force a locale that correctly interprets dates sent by Collect, unlike en_US" is a really helpful memory aide that doesn't require context switching.

Reply:
Done.